Kernel/System/Stats/Dynamic/TicketList.pm

```diff
@@ -949,11 +949,14 @@ sub GetStatTable {
                 next DYNAMICFIELD if !$IsStatsCondition;
 
                 # get new search parameter
-                my $DynamicFieldStatsSearchParameter = $DynamicFieldBackendObject->StatsSearchFieldParameterBuild(
-                    DynamicFieldConfig => $DynamicFieldConfig,
-                    Value              => $Param{Restrictions}->{$ParameterName},
-                    Operator           => $Operator,
-                );
+                my $DynamicFieldStatsSearchParameter;
+                if ($Param{Restrictions}->{$ParameterName}) {
+                    $DynamicFieldStatsSearchParameter = $DynamicFieldBackendObject->StatsSearchFieldParameterBuild(
+                        DynamicFieldConfig => $DynamicFieldConfig,
+                        Value              => $Param{Restrictions}->{$ParameterName},
+                        Operator           => $Operator,
+                    );
+                }
```

Comment:
Hi @pboguslawski 

we could just jump to the next dynamic field if the parameter is not present. 
Or do you see a problem there?


```suggestion
	            next DYNAMICFIELD if !$Param{Restrictions}->{$ParameterName};
	            
                my $DynamicFieldStatsSearchParameter = $DynamicFieldBackendObject->StatsSearchFieldParameterBuild(
                    DynamicFieldConfig => $DynamicFieldConfig,
                    Value              => $Param{Restrictions}->{$ParameterName},
                    Operator           => $Operator,
                );
```

Reply:
Your algo seems to work ok and is more elegant so applied in 33c0e56a931da03ec9a85f4d0a4791653750bfaf.